packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -79,12 +88,20 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       theme: theme!
     });
     const divButtonProps = buttonProps ? getNativeProps<React.HTMLAttributes<HTMLDivElement>>(buttonProps, divProperties) : undefined;
-
+    const theButton = (
+      <span
+        ref={this._thumb}
+        className={classNames.thumb}
+        style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)}
+        id={this._buttonId}
+        aria-labelledby={showThumbTooltip ? this._hostId : undefined}
+      />
+    );
     return (
       <div className={classNames.root}>
         {label && (
           <Label className={classNames.titleLabel} {...(ariaLabel ? {} : { htmlFor: this._id })} disabled={disabled}>
-            {label}
+            {label} : {value}
```

Comment:
Might consider still using the valueLabel formatting method here like how it was used when this used to be on the right side of the slider.